app/src/main/java/com/mychefassistant/framework/modules/DatabaseModule.kt

```diff
@@ -10,7 +10,7 @@ val databaseModule = module {
         Room.databaseBuilder(
             androidApplication(),
             ChefAssistantDatabase::class.java,
-            "ca_v4.db"
+            "ca_v5.db"

```

Comment:
The name of the db is only updated after a (Major) release. When you update the db schema between releases, there is no need to update the db version or name.
In general db name and db version is updated when a breaking (Major) change of schema happens which renders the old schema obsolete, and there is a need for migration of data, for instance You want to migrate user's data to the new schema without user noticing.

Reply:
Keep it at v1 before we release.